### PR TITLE
Add resource-level meta docs. Update top-level meta docs.

### PR DESCRIPTION
**Pull request by remear**
#### Purpose

Improve meta documentation.

#### Changes

- Add resource-level meta docs
- Clarify top-level meta docs

#### Related GitHub issues

#1235

### CODE REVIEW

File: docs/general/rendering.md [hunk 1]
```diff
@@ -81,8 +81,12 @@ PR please :)
 
 #### meta
 
-If you want a `meta` attribute in your response, specify it in the `render`
-call:
+A `meta` member can be used to include non-standard meta-information. `meta` can
```
Comment (bf4) [inline docs/general/rendering.md]: `s/non-standard/custom`

File: docs/general/rendering.md [hunk 2]
```diff
@@ -81,8 +81,12 @@ PR please :)
...
+be utilized in several levels in a response.
```
Comment (bf4) [inline docs/general/rendering.md]: `utilized` vs. `used`, go!

File: docs/general/rendering.md [hunk 3]
```diff
@@ -94,12 +98,33 @@ The key can be customized using `meta_key` option.
 render json: @post, meta: { total: 10 }, meta_key: "custom_meta"
 ```
 
-`meta` will only be included in your response if you are using an Adapter that supports `root`,
-as JsonAPI and Json adapters, the default adapter (Attributes) doesn't have `root`.
+`meta` will only be included in your response if you are using an Adapter that
+supports `root`, e.g., `JsonApi` and `Json` adapters. The default adapter,
+`Attributes` does not have `root`.
```
Comment (bf4) [inline docs/general/rendering.md]: Might want to collaborate with https://github.com/rails-api/active_model_serializers/pull/1557 in some way

File: docs/general/rendering.md [hunk 4]
```diff
@@ -94,12 +98,33 @@ The key can be customized using `meta_key` option.
...
+
+```ruby
+meta do
+  {
+    rating: 4,
+    comments_count: object.comments.count
+  }
+end
+```
+
```
Comment (bf4) [inline docs/general/rendering.md]: there's also jsonapi toplevel meta https://github.com/rails-api/active_model_serializers/blob/8981683b9fa8d0bcff0e76b3702d699689b787b4/docs/general/configuration_options.md#json-api

File: docs/general/rendering.md (hunk 1 above)
Comment (remear) [inline docs/general/rendering.md]: Sure, although that's copy from the JSON API docs.

Comment (bf4) [inline docs/general/rendering.md]: You're right. I saw that later :embarrassed:

File: docs/general/rendering.md (hunk 3 above)
Comment (remear) [inline docs/general/rendering.md]: My plan was to make minimal changes here so #1557 still has things that can be done.

File: docs/general/rendering.md (hunk 2 above)
Comment (remear) [inline docs/general/rendering.md]: Are you saying you'd like this changed to `used`?

File: docs/general/rendering.md (hunk 4 above)
Comment (remear) [inline docs/general/rendering.md]: Is there something specific you're direction my attention here to do?

**Comment by NullVoxPopuli** [comment]
LGTM, i'll merge, unless @bf4 has actionable comments

**Comment by bf4** [comment]
Haven't had time to review.  I trust it's an improvement, which is the most important thing.  First, do no harm and all.  

B mobile phone

**Comment by NullVoxPopuli** [comment]
looks good then, further changes, could be in an additional PR. :-)

**Pull request merged**